pkg/multicluster/local/local_cluster_manager_test.go

```diff
@@ -0,0 +1,49 @@
+package local
+
+import (
+	"github.com/ibm/the-mesh-for-data/pkg/multicluster"
+	"github.com/onsi/gomega"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"testing"
+)
+
+func TestLocalClusterManager(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+	s := scheme.Scheme
+	objs := []runtime.Object{
+		&v1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "cluster-metadata",
+				Namespace: "m4d-system",
+			},
+			Data: map[string]string{
+				"ClusterName": "remote-cluster",
+				"Region":      "Region-1",
+				"Zone":        "Zone-1",
+			},
+		},
+	}
+	cl := fake.NewFakeClientWithScheme(s, objs...)
+
+	cm := CreateLocalClusterManager(cl)
+	var actualClusters []multicluster.Cluster
+	var err error
+	if actualClusters, err = cm.GetClusters(); err != nil {
+		t.Error("TestMultiCluster failed")
```

Comment:
Too generic. Consider something like `t.Errorf("unexpected error in GetClusters: %v", err)`